TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/test/java/org/opensearch/migrations/transform/Es2OsPost23Test.java

```diff
@@ -53,6 +65,15 @@ private static Map<String,Object> parseJsonFromResourceName(String resourceName)
         }
     }
 
+    private static void transformJMESAndVerifyResult(Map<String,Object> json, String expectedValueSource) throws Exception{
+        var jmesTransformer = getJMESTransformer(json);
```

Comment:
These tests much be updated - they should send a config value (which should be null, since the user shouldn't have anything to configure) - NOT the document.